Source/Example Measurements/Keysight 34401A DMM Measurement/Keysight 34401A DMM Measurement.lvproj

```diff
@@ -117,20 +117,22 @@
 				<Property Name="Bld_excludeInlineSubVIs" Type="Bool">true</Property>
 				<Property Name="Bld_excludeLibraryItems" Type="Bool">true</Property>
 				<Property Name="Bld_excludePolymorphicVIs" Type="Bool">true</Property>
-				<Property Name="Bld_localDestDir" Type="Path">../builds/NI_AB_PROJECTNAME/Keysight 34401A DMM Measurement</Property>
-				<Property Name="Bld_localDestDirType" Type="Str">relativeToCommon</Property>
+				<Property Name="Bld_localDestDir" Type="Path">/c/build_output/keysight</Property>
```

Comment:
I'm not really sure how this works in the LV editor, but it would be nice to continue to use `NI_AB_PROJECTNAME` here rather than "keysight" as the folder name. This would be consistent with the Game Of Life project above.